src/client/src/views/Article.vue

```diff
@@ -76,32 +91,68 @@
          currentArticleId: this.$route.params.id,
          currentArticleOutlet: this.$route.params.outlet,
-      }
+         apiRoot: process.env.VUE_APP_API_ROOT,
+      };
     },
     computed: {
       currentArticle: function(){
         if (store.state.articles && Object.keys(store.state.articles).length > 0) {
           return store.state.articles[this.currentArticleOutlet][this.currentArticleId]
```

Comment:
SITUATION: i created a comment with user1 on an article. with user2, i open up that same article and the comment isn't there. 

this might be a problem with just creating an axios call to the back-end without also updating the store. i'd suggest doing both

Reply:
actually, it might be a problem with just grabbing the article from the store. the best way seems to be through a GET request

Reply:
discussed in sprint planning